migrations/versions/774d0d537e69_update_key_length_on_image.py

```diff
@@ -0,0 +1,41 @@
+"""update key length on image
```

Comment:
This shouldn't cause any issues with previous keys - at this level of the system, these are simply strings, and we are expanding the max length of the possible strings. This did not cause any issues before because 64 was plenty of space for `["check", "project_media", project_media.id, field_name]`, base-64 encoded, which is what all strings were historically. In this update, we needed to add the idea of a "temporary" ID which is just a SHA256 of the URL (see the line `project_media.id = Digest::MD5.hexdigest(project_media.url).to_i(16)` in Check API / `alegre_v2.rb`). The size of that ID is, what, 33 orders of magnitude larger or so than our current IDs, so its just such a large (fake and temporary but technically an integer that will also never collide) number that it forces us to expand the string space. These are short-lived objects, but while alive we need to accommodate their ridiculous strings. There are probably better ways of re-architecting lots of this, but we're already on a side-quest from the alrger goal and I wanted to figure out a straightforward way to keep us as on track as ossible.